sql/01-bde_control_tables.sql

```diff
@@ -92,21 +96,46 @@ GRANT SELECT ON TABLE upload_table TO bde_user;
 
 COMMENT ON TABLE upload_table IS
 $comment$
-Tracks the status of uploads for each table
-table_key_column is unique non-composite, not null integer or bigint column used for identifying the table row for incremental updates. This identifier must be the same as defined in the cbe_tables.tablekeycolumn
-last_upload_id is the id of the last upload job affecting this table
-last_upload_dataset is the dataset id of the last level 5 or 0 uploaded (since the level 0 upload will override any level 5 uploads that have been applied).
-last_upload_type is either 0 or 5
-last_upload_incremental is true if the table data was updated, and false if the table data was completely refreshed from a level 0
+Tracks the status of uploads for each table.
+
+key_column is the name of a unique non-composite, not null
+integer or bigint column used for identifying the table row for
+incremental updates. This identifier must be the same as defined in
+the cbe_tables.tablekeycolumn (?).
+
+last_upload_id is the id of the last upload job affecting this table,
+referencing the id field of the upload table.
+
+last_upload_dataset is the dataset id of the last level 5 or 0 uploaded
+(since the level 0 upload will override any level 5 uploads that have
+been applied).
+
+last_upload_type is either 0 or 5.
+
+last_upload_incremental is true if the table data was updated, and
+false if the table data was completely refreshed from a level 0.
+
 last_upload_details is a text string with details of the last upload
-   (currently will contain constituent files and end times, for checking L5 uploads against.)
-last_upload_time records when the upload was applied
-last_upload_bdetime is a timestamp for the last BDE upload applied
-last_level0_dataset is the dataset id of the last level 0 uploaded
+(currently will contain constituent files and end times, for checking
+L5 uploads against).
+
+last_upload_time records when the table upload was started.
+
+last_upload_bdetime is the timestamp found in the last BDE file
+uploaded.
+
+last_level0_dataset is the dataset id of the last level 0 uploaded.
+
 upl_id_lock is the id of an upload currently locking the table (this is
-  in the sense of a process lock, not a database lock)
-row_tol_warning is the minimum ratio of the number of number of rows in the table after a level 0 update  to the number before the update before a warning is logged
-row_tol_error is the minimum ratio of the number of number of rows in the table after a level 0 update  to the number before the update before an exception is thrown
+in the sense of a process lock, not a database lock).
+
+row_tol_warning is the minimum ratio of the number of number of rows
```

Comment:
Ah yes those words are pretty mixed up in terms of grammar and looking at the parameter definition and what it's doing it pretty confusing. What was I on when I wrote this? Definitely needed a peer review!

The parameter is a maximum percentage ratio between the current table row count and the new table row count after an incremental level_0 update before a warning/error is raised. So a value of 0.95 a would mean the less than a 5% row count change is allowed before a warning/error is raised. The reason for these parameters is due to past operational issues we had with Landonline dumping out partial BDE exports that only contained a small number of the table's total rows, but still had a valid headers.